docs/articles/basics.md

```diff
@@ -53,9 +55,13 @@ To run an interactive shell in the Ubuntu image:
 
     $ docker run -i -t ubuntu /bin/bash       
   
-The `-i` flag starts an interactive container. The `-t` flag creates a pseudo-TTY that attaches `stdin` and `stdout`.  
+The `-i` flag starts an interactive container. The `-t` flag creates a
+pseudo-TTY that attaches `stdin` and `stdout`.  
 
-To detach the `tty` without exiting the shell, use the escape sequence `Ctrl-p` + `Ctrl-q`. The container will continue to exist in a stopped state once exited. To list all containers, stopped and running use the `docker ps -a` command.
+To detach the `tty` without exiting the shell, use the escape sequence
+`Ctrl-p` + `Ctrl-q`. The container will continue to exist in a stopped state
+once exited. To list all containers, stopped and running use the `docker ps -a`
```

Comment:
Reading this, I think there should be a comma after "stopped and running" (not your change, just noticed it here)